src/main/java/spark/Spark.java

```diff
@@ -0,0 +1,92 @@
+package spark;
+
+import spark.exceptions.SparkException;
+import spark.parser.Parser;
+import spark.parser.commands.commandtypes.ExitCommand;
+import spark.storage.Storage;
+import spark.tasks.TaskList;
+import spark.parser.commands.commandtypes.Command;
+import spark.parser.commands.commandtypes.ListCommand;
+import spark.exceptions.fileexceptions.FileException;
+import spark.exceptions.fileexceptions.TaskDecodingException;
+
+public class Spark {
+    private static final String defaultFilePathString = "spark_save_file.txt";
+    private TaskList taskList;
+    private Ui ui;
+    private Storage storage;
+
+    /**
+     * Starts an instance of Spark that stores saved Tasks in
+     * the default relative file-path on the user's hard-disk.
+     */
+    public Spark() {
+        this.ui = new Ui();
+
+        try {
+            this.storage = new Storage(defaultFilePathString);
+            this.taskList = new TaskList(storage.readTasksFile());
+        } catch (FileException | TaskDecodingException e) {
+            ui.printException(e);
+            ui.printMessageWithDivider("Note that any changes will not be saved to your save-file.");
+            this.taskList = new TaskList();
+        }
+    }
+
+    /**
+     * Starts an instance of Spark that stores saved Tasks in
+     * the specified relative file-path on the user's hard-disk.
+     */
+    public Spark(String filePathString) {
+        this.ui = new Ui();
+
+        try {
+            this.storage = new Storage(filePathString);
+            this.taskList = new TaskList(storage.readTasksFile());
+        } catch (FileException | TaskDecodingException e) {
+            ui.printException(e);
```

Comment:
You may consider including the same exception message(line 31) here to be consistent 😄